avocado/core/decorators.py

```diff
@@ -53,39 +53,3 @@ def wrap(*args, **kwargs):
     if func:
         return decorate(func)
     return decorate
-
-
-def skip(message=None):
-    """
-    Decorator to skip a test.
-    """
-    def decorator(function):
-        if not isinstance(function, type):
-            @wraps(function)
-            def wrapper(*args, **kwargs):
-                raise core_exceptions.TestDecoratorSkip(message)
-            function = wrapper
-        return function
-    return decorator
-
-
-def skipIf(condition, message=None):
-    """
-    Decorator to skip a test if a condition is True.
-    """
-    if condition:
-        return skip(message)
-    return _itself
-
-
-def skipUnless(condition, message=None):
-    """
-    Decorator to skip a test if a condition is False.
-    """
-    if not condition:
-        return skip(message)
-    return _itself
-
-
-def _itself(obj):
-    return obj
```

Comment:
Well my point is that I'd prefer re-using as much from `unittest` as possible, because it's proven to work well and users expect it to behave that way. For this reason I'd prefer us to try to adjust avocado in order to be more-like unittest (this is just a draft to start the discussion, not a polished version).